vue/sbc-common-components/src/components/SbcFeeSummary.vue

```diff
@@ -71,6 +71,9 @@ export default class SbcFeeSummary extends Vue {
   @Prop({ default: '' })
   private payURL!: string
 
+  @Prop()
+  private filingLabel!: string
```

Comment:
FYI, it's usually a good idea to provide a default value (as on line 71) so it's never undefined.

If you're concerned about being able to identify an empty string, all of these are "falsy":
- filingLabel is undefined
- filingLabel is null
- filingLabel is ''

That's why you can use use a simple expression to check if filingLabel is valid.

No change needed.